### PR TITLE
Aden 2983 unused slots

### PR DESCRIPTION
**Pull request by jacekjursza**
Cleaning up code after reviewing unused ad - slots.

### CODE REVIEW

File: extensions/wikia/Lightbox/scripts/Lightbox.js [hunk 1]
```diff
@@ -855,7 +846,7 @@
 			var types = Lightbox.carouselTypes,
 				deferredList = [],
 				// show-ads class appears when there is going to be a MODAL_RECTANGLE ad
```
Comment (drets) [inline extensions/wikia/Lightbox/scripts/Lightbox.js]: Should we remove comment above?

File: extensions/wikia/Lightbox/scripts/Lightbox.js [hunk 2]
```diff
@@ -76,13 +76,6 @@
 			// cache re-used DOM elements and templates for this modal instance
 			Lightbox.cacheDOM();
 
-			// Init ads in Lightbox
-			if ($('#MODAL_RECTANGLE').length && Lightbox.ads.showAds()) {
-				Lightbox.openModal.lightbox.addClass('show-ads');
```
Comment (drets) [inline extensions/wikia/Lightbox/scripts/Lightbox.js:81]: Maybe we also need to remove: https://github.com/Wikia/app/blob/ADEN-2983-unusedSlots/extensions/wikia/Lightbox/styles/Lightbox.scss#L402-L435 ?

**Comment by drets** [comment]
I am not sure if it's worth to touch old liftium and remove those guys: https://github.com/Wikia/app/blob/ADEN-2983-unusedSlots/extensions/wikia/AdEngine/js/provider/liftium.js#L24-L25 ?

File: extensions/wikia/Lightbox/scripts/Lightbox.js (hunk 2 above)
Comment (jacekjursza) [inline extensions/wikia/Lightbox/scripts/Lightbox.js:81]: I'm not sure. Aren't it used for the second ad in the lightbox?

**Comment by jacekjursza** [comment]
Maybe let's wait for it with full Liftium cleanup?

File: extensions/wikia/Lightbox/scripts/Lightbox.js (hunk 1 above)
Comment (jacekjursza) [inline extensions/wikia/Lightbox/scripts/Lightbox.js]: Good catch, thx ;)

**Comment by drets** [comment]
I agree that we can wait for full Liftium cleanup but I am curious what @nandy-andy thinks about it :wink:

File: extensions/wikia/Lightbox/scripts/Lightbox.js (hunk 2 above)
Comment (drets) [inline extensions/wikia/Lightbox/scripts/Lightbox.js:81]: I'd say it was used only by `MODAL_RECTANGLE` but, definitely, it's worth to do happy-path testing (after removing those styles).

**Comment by drets** [comment]
If we shouldn't remove styles which may be related to `MODAL_RECTANGLE` rectangle ad - LGTM :+1:

**Comment by drets** [comment]
LGTM :+1:

**Pull request merged**